### PR TITLE
Updated danish GUI translation

### PR DESCRIPTION
**Pull request by cryptobench**

### CODE REVIEW

**Comment by ordtrogen** [comment]
image

**Comment by erciccione** [comment]
ref #1377

**Comment by erciccione** [comment]
please rebase using this commands:

`git pull upstream master --rebase`

then

`git checkout --theirs translations/monero-core_da.ts && git add --all && git rebase --continue && git push -f origin master`

**Comment by cryptobench** [comment]
God dammit email app signed me out again. Sorry for the delay..

**Comment by cryptobench** [comment]
Everything should be fixed now @ordtrogen  @erciccione . Thanks for the help :)

File: translations/monero-core_da.ts [hunk 1]
```diff
@@ -980,24 +980,24 @@
     <message>
         <location filename="../pages/Settings.qml" line="207"/>
         <source>Daemon mode</source>
-        <translation type="unfinished"></translation>
+        <translation>Daemon mode</translation>
```
Comment (ordtrogen) [inline translations/monero-core_da.ts]: 'Mode', isn't that 'tilstand' in Danish?

File: translations/monero-core_da.ts [hunk 2]
```diff
@@ -1662,32 +1662,32 @@ The old wallet cache file will be renamed and can be restored later.
...
         <location filename="../pages/Transfer.qml" line="46"/>
         <source>&lt;style type=&apos;text/css&apos;&gt;a {text-decoration: none; color: #FF6C3C; font-size: 14px;}&lt;/style&gt;&lt;font size=&apos;2&apos;&gt; (&lt;/font&gt;&lt;a href=&apos;#&apos;&gt;Start daemon&lt;/a&gt;&lt;font size=&apos;2&apos;&gt;)&lt;/font&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;style type=&apos;text/css&apos;&gt;a {text-decoration: none; color: #FF6C3C; font-size: 14px;}&lt;/style&gt;&lt;font size=&apos;2&apos;&gt; (&lt;/font&gt;&lt;a href=&apos;#&apos;&gt;Start daemon&lt;/a&gt;&lt;font size=&apos;2&apos;&gt;)&lt;/font&gt;</translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="82"/>
         <source>Ring size: %1</source>
-        <translation type="unfinished"></translation>
+        <translation>Ring størrelse: %1</translation>
```
Comment (ordtrogen) [inline translations/monero-core_da.ts]: It looks like you're practicing "Særskrivning" in many places in the translation. 
Here, for example, shouldn't it be "Ringstørrelse"?

File: translations/monero-core_da.ts [hunk 3]
```diff
@@ -1733,12 +1733,12 @@ The old wallet cache file will be renamed and can be restored later.
     <message>
         <location filename="../pages/Transfer.qml" line="433"/>
         <source>Advanced options</source>
-        <translation type="unfinished"></translation>
+        <translation>Avanceret indstillinger</translation>
```
Comment (ordtrogen) [inline translations/monero-core_da.ts]: "Avancerede"

File: translations/monero-core_da.ts [hunk 4]
```diff
@@ -2159,7 +2161,7 @@ For the case of outgoing payments, you can get a &apos;Spend Proof&apos; that pr
     <message>
         <location filename="../wizard/WizardFinish.qml" line="77"/>
         <source>Network Type</source>
-        <translation type="unfinished"></translation>
+        <translation>Netværks Type</translation>
```
Comment (ordtrogen) [inline translations/monero-core_da.ts]: "Netværkstype"?

File: translations/monero-core_da.ts [hunk 5]
```diff
@@ -2498,70 +2501,70 @@ Fee: </source>
...
-        <translation type="unfinished">Ringstørrelse:</translation>
+        <translation>Ringstørrelse:</translation>
     </message>
     <message>
         <location filename="../main.qml" line="627"/>
         <source>
 
 WARNING: non default ring size, which may harm your privacy. Default of 7 is recommended.</source>
-        <translation type="unfinished"></translation>
+        <translation>ADVARSEL: ikke standard ring størrelse, som kan skade din anonymitet. Standarden 7 er anbefalet.</translation>
```
Comment (ordtrogen) [inline translations/monero-core_da.ts]: "ringstørrelse"?

File: translations/monero-core_da.ts [hunk 6]
```diff
@@ -1218,155 +1218,155 @@ The old wallet cache file will be renamed and can be restored later.
...
         <location filename="../pages/SharedRingDB.qml" line="346"/>
         <source>Set Ring</source>
-        <translation type="unfinished"></translation>
+        <translation>Sæt Ring</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="364"/>
         <source>I intend to spend on key-reusing fork(s)</source>
-        <translation type="unfinished"></translation>
+        <translation>Jeg har hensigt at bruge på en nøgle-genbrugene fork(s)</translation>
```
Comment (ordtrogen) [inline translations/monero-core_da.ts]: Do you use plural 's' in Danish?   I think it works well to just do "fork" and leave out the optional plural

**Comment by cryptobench** [comment]
@ordtrogen You're probably better at danish than I am myself. Thanks for pointing those out. I will fix them right now.

**Comment by ordtrogen** [comment]
I don't even speak Danish although I do understand a fair bit from watching Klovn :-) Your name sounds Danish though.

**Comment by cryptobench** [comment]
Haha! Great way of learning danish :) 
Pushing fixes now.

**Comment by erciccione** [comment]
@cryptobench please squash to one single commit when done. Also, there is a merge commti tha shouldn't be there

**Comment by cryptobench** [comment]
Wooppps dunno what I did there with the "
Merge branch 'master' of https://github.com/cryptobench/monero-gui"

**Comment by cryptobench** [comment]
How do I revert? haha

**Comment by cryptobench** [comment]
Ugh the merge branch comes after Im trying to squash

**Comment by cryptobench** [comment]
I really messed up now. Will stop touching stuff - looking for help

**Comment by erciccione** [comment]
`git reset --hard HEAD~1` will remove that merge commit. Let me know if you need help from that point

**Comment by cryptobench** [comment]
Yes more help please :p

**Comment by erciccione** [comment]
@cryptobench I suggest you to backup trhe updated file, use `git reset HEAD~1` to remove the merge commit and the empty commits with it. Then paste the backup file on the old one, recommit then rebase. At that point you will have 2 commits on this branch. use `git rebase -i HEAD~2` to rebase and squash. This way we should avoid any merge commit.

**Comment by erciccione** [comment]
@cryptobench add me as a collaborator to your repo, will fix that for you. We are running out of time for the release

**Comment by cryptobench** [comment]
Invite sent

**Pull request merged**